doc/man/8/ceph.rst

```diff
@@ -174,10 +176,87 @@ Usage::
 	ceph compact
 
 
+config
+------
+
+Configure the cluster. By default, Ceph daemons and clients retrieve their
+configuration options from monitor when they start, and are updated if any of
+the tracked options is changed at run time. It uses following additional
+subcommand.
+
+Subcommand ``dump`` to dump all options for the cluster
+
+Usage::
+
+	ceph config dump
+
+Subcommand ``ls`` to list all option names for the cluster
+
+Usage::
+
+	ceph config ls
+
+Subcommand ``help`` to describe the specified configuration option
+
+Usage::
+
+    ceph config help <option>
+
+Subcommand ``get`` to dump the option(s) for the specified entity.
+
+Usage::
+
+    ceph config get <who> {<option>}
+
+Subcommand ``set`` to set an option for one or more specified entities
```

Comment:
Missing the `show` subcommand, which shows the runtime values that a daemon is currently reporting.

Reply:
added `ceph config show` and `ceph config show-with-defaults`.